components/resource-creation/ResourcePanel.tsx

```diff
@@ -65,7 +65,7 @@ export default function ResourcePanel() {
                   measureBundle.content,
                   measurementPeriod.start?.toISOString(),
                   measurementPeriod.end?.toISOString(),
-                  useTrustMetaProfile
+                  trustMetaProfile
```

Comment:
Same comment as above.

Reply:
Never mind! Github was showing me an out of date copy of `calculateDetailedResult`.